server/templates/Chart.template.js

```diff
@@ -0,0 +1,20 @@
+import basePathConfig from '../configs/basePathConfig';
+
+const basePath = basePathConfig || '';
+
+export default () => {
+  return `
+		<!DOCTYPE html>
```

Comment:
can we fix this formatting?

Reply:
[2ac7584](https://github.com/AMP-SCZ/dpdash/pull/76/commits/2ac7584ace0ff2a0b54d94798b482358dd393cb0)